jitsi_meet_wrapper/example/android/app/build.gradle

```diff
@@ -55,6 +55,8 @@ android {
             // TODO: Add your own signing config for the release build.
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
+            minifyEnabled true
```

Comment:
Why is this required? I added the proguard files to the example in e7c47ada24bc9bd16b43ea83c3b627fbcd113953 and it also works without this.